app/Http/Controllers/Api/V1/OrganizationController.php

```diff
@@ -55,4 +56,12 @@ public function getByDomain(OrganizationRequest $organizationRequest)
             'organization' => new OrganizationResource($this->organizationRepository->findByField('domain', $data['domain'])),
         ], 200);
     }
+
```

Comment:
Please add doc comments 
and as mentioned by @aqeel-tkxel consider using Request Validation.